src/theme/SearchBar/lib/lunar-search.js

```diff
@@ -1,4 +1,4 @@
-import lunr from "@generated/lunr.client";
+import * as lunr from "lunr";
```

Comment:
Also please revert this changes too - related to lanuage support

Reply:
Was not intend, only the first commit. My mistake - i needed those changes at the moment for Webpack 5 support.